addons/binding/org.openhab.binding.mihome/README.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: documentation
+---
+
+{% include base.html %}
+
+# Xiaomi Mi Smart Home Binding
+
+The binding allows your openHAB to communicate with Xiaomi Smart Home Suite. 
+
+In order to connect the Gateway, you need to install MiHome app 
+from the [Google Play](https://play.google.com/store/apps/details?id=com.xiaomi.smarthome) or [AppStore](https://itunes.apple.com/app/mi-home-xiaomi-for-your-smarthome/id957323480).
+
+## How to set up
+1. Install the binding
+2. Setup gateway to be discoverable
+   1. Add Gateway 2 to your WiFi Network
+   2. Install Mi Home app from [Google Play](https://play.google.com/store/apps/details?id=com.xiaomi.smarthome) or [AppStore](https://itunes.apple.com/app/mi-home-xiaomi-for-your-smarthome/id957323480)
+   3. Set your region to Mainland China under Settings -> Locale (seems to be required)
+   4. Update gateway (maybe multiple times)
+   5. Enable developer mode:<br />
+      1. Select your Gateway in Mi Home
+      2. Go to the "..." menu and click "About"
+      3. Tap the version number "Version : 2.XX" at the bottom of the screen repeatedly until you enable developer mode
+      4. you should now have 2 extra options listed in Chinese.
+      5. Choose the first new option (fourth position in the menu or the longer text in Chinese)
+      6. Tap the toggle switch to enable LAN functions. Note down the developer key (something like: 91bg8zfkf9vd6uw7).
+      7. Make sure you hit the OK button (to the right of the cancel button) to save your changes.
+4. In openHAB you should now be able to discover the Xiaomi Gateway
+5. Enter the previously noted developer key in openHAB Paper UI -> Configuration -> Things -> Xiaomi Gateway -> Edit -> Developer Key. Save.
+   (This is required if you want to be able to send controls to the devices like the light of the gateway.)
+6. Your sensors should be getting discovered by openHAB as you add and use them.
+
+## Connecting sub-devices (sensors) to the Gateway
+There are two ways of connecting Xiaomi devices to the gateway:
+1. Online - within the MiHome App
+2. Offline - manual
+    * Click 3 times on the Gateway's button
```

Comment:
[Something broke here](https://github.com/carecon/openhab2-addons/tree/master/addons/binding/org.openhab.binding.mihome#connecting-sub-devices-sensors-to-the-gateway) probably due to lack of spacing.